### DISCUSSION TITLE
How to Earn Achievements on GitHub: Seeking Tips and Sharing My Journey!

### DISCUSSION THREAD

**Discussion by sameer266** [community/community#147998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I’m excited to be actively engaging with the GitHub community and am working toward earning achievements here. GitHub has always been a great place for collaboration, and I’ve been exploring various features like pull requests, issues, and contributions. But now, I’m diving into Discussions to connect with even more users and learn from their experiences.

💡 My goal: I want to start meaningful conversations and discussions around coding, development practices, and best practices for maintaining repositories. I'd love to hear your tips for how to make the most out of Discussions on GitHub.

Have you achieved the Discussions badge or other related accomplishments? How did you approach creating engaging discussions that contribute to the community?

Looking forward to hearing your thoughts and learning from your experiences. Let’s grow together!

#GitHubAchievements #CommunityEngagement #LearningTogether #Discussions #OpenSource

**Comment by mecodeatlas** [comment]
Thanks so much for posting in our Community @sameer266. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
